README.md

```diff
@@ -0,0 +1,10 @@
+# Gigantic Parser Module
+
+A Module to Parse Data from *Motiga's* **Gigantic**
+
+
+## Installation:
+
+```
+pip install -e .[development]
```

Comment:
When used with the `-e` flag, `pip install` installs the package in 'editable mode' which you only really want to use in development. `pip install .` is a perfectly good install command outside of development though, and will install dependencies listed in the setup.py file.